lib/models/device.coffee

```diff
@@ -895,6 +905,10 @@ getDeviceModel = (deps, opts) ->
 	# @function
 	# @memberof resin.models.device
 	#
+	# @deprecated
+	# This is not supported on multicontainer devices, and will
+	# be removed in future
```

Comment:
NIT: `removed in future` --> `removed in the future` or `removed in a future release`